app/main.py

```diff
@@ -1,8 +1,22 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> dict:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = []
+
+    for person in people:
+        Person(person["name"], person["age"])
+    for person in people:
+        individual = Person.people[person["name"]]
+        if person.get("wife") and person["wife"] is not None:
```

Comment:
Look:
```python
if person.get("wife") and person["wife"] is not None:
```
is the same:
```python
if person.get("wife"):
```
Read a theory about that. And change this condition (as the second one is unnecessary)